AutomatedTesting/Gem/PythonTests/Performance/TestSuite_Main_DX12.py

```diff
@@ -0,0 +1,34 @@
+"""
+Copyright (c) Contributors to the Open 3D Engine Project.
+For complete copyright and license terms please see the LICENSE at the root of this distribution.
+
+SPDX-License-Identifier: Apache-2.0 OR MIT
+
+"""
+
+# This suite consists of all test cases that are passing and have been verified.
+
+import pytest
+import os
+import sys
+
+from ly_test_tools import LAUNCHERS
+sys.path.append(os.path.dirname(os.path.abspath(__file__)) + '/../automatedtesting_shared')
+
+from base import TestAutomationBase
+
+@pytest.mark.SUITE_main
+@pytest.mark.parametrize("launcher_platform", ['windows_editor'])
+@pytest.mark.parametrize("project", ["AutomatedTesting"])
+class TestAutomation(TestAutomationBase):
```

Comment:
Strongly recommend using `ly_test_tools.o3de.editor_test.EditorTest` instead of `automatedtesting_shared.base.TestAutomationBase`, even if you are not batching or parallelizing tests.

Reply:
Tests were modeled after existing tests but noted and will update